include/highfive/bits/H5Node_traits_misc.hpp

```diff
@@ -358,6 +358,29 @@ inline void NodeTraits<Derivate>::createExternalLink(const std::string& link_nam
     }
 }
 
+template <typename Derivate>
+template <typename T, typename>
+inline void NodeTraits<Derivate>::createHardLink(const std::string& link_name,
+                                                 const T& target_obj,
+                                                 LinkCreateProps linkCreateProps,
```

Comment:
```suggestion
                                                 const LinkCreateProps& linkCreateProps,
```

Reply:
As in createSoftLink and createExternalLink, if parents is true, the link create properties have to be modified.
I tried to maintain consistent API with these 2 methods.

Reply:
thank you for explanation :)